### DISCUSSION TITLE
Hoe to create Profile-Groups

### DISCUSSION THREAD

**Discussion by mighty-services** [BornToBeRoot/NETworkManager#1424] [Q&A] [answered]
Dear Community,

I love this tool, but have quite some endpoints to watch. I know already how to create profiles for the endpoints, but need to sort them in several Groups. I don’t know how to create new groups beside the standard one, which is there from the start.

**Comment by BornToBeRoot** [comment, marked as answer]
Hello, you can simply enter a new name here:

![image](https://user-images.githubusercontent.com/16019165/165154774-0e6ca14e-6706-4968-b752-d445d91699c8.png)

The new group will be created when the profile is added.

OR

In the settings > profiles:

![image](https://user-images.githubusercontent.com/16019165/165155098-d82bc3e7-ac4f-4b36-ba18-caa83cd109ec.png)
